src/ILVerify/src/ILVerify.csproj

```diff
@@ -9,10 +9,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\Common\src\TypeSystem\IL\HelperExtensions.cs" Link="TypeSystem\IL\HelperExtensions.cs" />
+    <Compile Include="..\..\Common\src\TypeSystem\IL\Stubs\ILEmitter.cs" Link="TypeSystem\IL\Stubs\ILEmitter.cs" />
```

Comment:
`HelperExtensions.EmitCallThrowHelper` depends on it. But I can remove both files and reimplement `HelperExtensions.GetKnownType` in ILVerify.